Assets/Scripts/GrappScript.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "GrappScript", menuName = "LTG5/Weapons/GrappleHook", order = 0)]
+public class GrappScript : ProjectileWeapon
+{
+    SpringJoint2D joint;
+    Vector2 grapplePoint;
+    public LineRenderer lr;
+    
+    override
+    public void Fire(Vector2 direction)
+    {
+        lr = GameObject.FindGameObjectWithTag("Player").GetComponent<LineRenderer>();
```

Comment:
Is there going to be a "Player" object that has a line renderer?

Reply:
Maybe just create a new one when needed? The cost isn't that much and it keeps the player cleaner.